README.md

```diff
@@ -8,5 +8,65 @@
 # Atlan Go SDK
 This repository houses the code for a Go SDK to interact with [Atlan](https://atlan.com).
 
+
+## Installing for Development
+
+### Initial Setup
+To get started developing or using the SDK:
```

Comment:
Can you include this here? It links to our getting started guide in the developer docs, which can help anyone looking to use or develop the SDK:
`To get started developing or using the SDK, follow the steps below. For a more detailed overview, check out our [Getting Started Guide](https://developer.atlan.com/getting-started/)`